README.md

```diff
@@ -245,25 +247,26 @@ Usage:
   ec2-metadata-mock spot [--action ACTION] [flags]
 
 Aliases:
-  spot, spotitn
+  spot, spot
```

Comment:
this should stay as `spotitn` since it's a valid alias

Reply:
I think this was changed in this [commit](https://github.com/aws/amazon-ec2-metadata-mock/commit/7621e835590920881c85f81467f3df0df8ab7513#diff-9c3028e9a33c2fdb57e5fb8c9ced84dfL70) already. I just copy-pasted the help output for updating the readme

Reply:
ah, got it I'll look into it. I'll approve+merge once the travis checks pass!